src/components/menu/menu.spec.ts

```diff
@@ -376,6 +382,32 @@ describe('MenuComponent', () => {
 
         expect(component.item1Activated).toHaveBeenCalledTimes(1);
     });
+
+    it('should not emit a click event when disabled is true.', async () => {
+        component.disabled = true;
+        fixture.detectChanges();
+        await fixture.whenStable();
+
+        // open menu
+        component.trigger.openMenu();
+        fixture.detectChanges();
+        await fixture.whenStable();
+
+        // get the child submenu item
+        const menuItem = document.querySelector<HTMLButtonElement>(
+            '#menu-item-disabled'
+        );
+
+        expect(menuItem).toBeTruthy();
+        spyOn(component, 'onButtonClick');
+
+        // perform a click
+        menuItem.click();
+        fixture.detectChanges();
+        await fixture.whenStable();
+
+        expect(component.onButtonClick).not.toHaveBeenCalledWith();
```

Comment:
I believe this should be `toHaveBeenCalled`.

Reply:
Should it not be .not.toHaveBeenCalledWith(); because we don't want the event to be called when disabled is true or am I misunderstanding

Reply:
`toHaveBeenCalledWith` verifies that a function has been called with the provided arguments, e.g. `.toHaveBeenCalledWith(2)` would check that a function has been called with an argument of 2. You're not checking any arguments here, you just want to check that it has not been called at all with any kind of argument.